Sources/Misc/SandboxEnvironmentDetector.swift

```diff
@@ -37,4 +37,11 @@ final class DefaultSandboxEnvironmentDetector: SandboxEnvironmentDetector {
         return url.path.contains("sandboxReceipt")
     }
 
+    #if DEBUG
+    // Mutable in tests so it can be overriden
+    static var `default`: SandboxEnvironmentDetector = BundleSandboxEnvironmentDetector()
+    #else
+    static let `default`: SandboxEnvironmentDetector = BundleSandboxEnvironmentDetector()
```

Comment:
`var` vs `let` here is clever! FWIW just having it be internal would likely have been enough for testing purposes, but this adds extra safety